geo/src/utils.rs

```diff
@@ -98,7 +98,7 @@ where
     //
     // See: https://en.wikipedia.org/wiki/Point_in_polygon
 
-    assert!(linestring.is_closed());
+    debug_assert!(linestring.is_closed());
```

Comment:
I'm not sure if this is controversial - we have some "undefined behavior" for invalid geometries, yet we don't (yet) have a way of asserting validity of geometries. 

I think we should be a little more conservative about crashing in release builds.